### ISSUE TITLE
Initial FPS project builds forever

### ISSUE THREAD

**Issue by AndrewRayCode** [BabylonJS/Editor#321]
I downloaded the latest editor v4.0.4 and created a new FPS project with graph, and this never goes away: 

<img width="318" alt="Screen Shot 2021-10-16 at 10 40 05 PM" src="https://user-images.githubusercontent.com/79215/137613429-675d4940-181f-4ef4-b7e3-4dd55b1a3d45.png">

I don't know where to look for debugging, the last line of the Common tab is
> [INFO]: Server is running.

**Comment by julien-moreau** [comment]
@AndrewRayCode any news?

**Comment by AndrewRayCode** [comment]
I haven't tried re-opening the editor since that event. If i open it now and create a new project, it was stuck in "updating dependencies" for about 10 minutes with no progress bar movement, now it's been building the project for quite some time, and also it was pretty easy to miss the popup of "babylonjs wants to access your ___ folder" where the project drive was chosen, since it seems pretty delayed (10-15 minutes from when I chose the folder in finder). Then it took about 5 minutes to fully compile. After what seems like 15 minutes, maybe my scene is ready now?

However when I clicked play, I got the error:

> [ERROR]: An error happened: Cannot find module '/Users/me/Documents/Projects/BabylonEditorAndyScenes/blorf/build/src/scenes/scene/index.js'
Require stack:
> - /Applications/BabylonJS Editor.app/Contents/Resources/app.asar/build/src/renderer/play/inline-play.js
> - /Applications/BabylonJS Editor.app/Contents/Resources/app.asar/build/src/renderer/editor/components/preview.js
> - /Applications/BabylonJS Editor.app/Contents/Resources/app.asar/build/src/renderer/editor/editor.js
> - /Applications/BabylonJS Editor.app/Contents/Resources/app.asar/build/src/renderer/editor/index.js
> - /Applications/BabylonJS Editor.app/Contents/Resources/app.asar/html/editor.html
 
And now there's a progress bar stuck at "Compiling (compiling): 0%"

I'm unable to reproduce the original issue, and the new one appears to be long build times, lacking UX for folder permissions, and getting stuck on the compiling step. This could be due to me not giving additional folder permissions that are required, I'm not sure. My fans are at full blast, I'm not sure what the issue is. I don't plan to do any additional debugging on this.
